src/condor_utils/docker-api.cpp

```diff
@@ -101,21 +103,34 @@ int DockerAPI::createContainer(
 	runArgs.AppendArg( "--cidfile=" + cidFileName );
 */
 
-	
+
 	// Configure resource limits.
-	
-	// First cpus
-	int  cpus;
```

Comment:
My only question is: do the "cpuset-cpus" and "cpu-shares" features are mutually exclusive?
Cause we treat them that way. if not we can leave the the previous code (cpu-shares) as-is and just add the cpu-shares code block.